src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/AssessmentOrgsRepository.cs

```diff
@@ -90,36 +93,32 @@ public IEnumerable<Organisation> GetOrganisationsByStandardId(string standardId)
                 throw new ApplicationException($"Failed query organisations by standard id");
             }
 
-            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date);
+            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date && (x.EffectiveTo >= DateTime.UtcNow.Date || x.EffectiveTo == null)).ToList();
```

Comment:
i will add an excel tests to check for an empty effective from date. so we don't have to carry the validation in our api.